### DISCUSSION TITLE
problem in  make deploy-sepolia

### DISCUSSION THREAD

**Discussion by Suhas-30** [Cyfrin/foundry-full-course-cu#2247] [Q&A] [answered]
In this course they made folders for test's. I also did same thing but now  I'm finding  error 
error in below
: error="/mnt/c/Users/LENOVO/Documents/Foundry/Fund-ME/test/FundMe.t.sol": No such file or directory (os error 2)    
Error: 
failed to read artifact source file for `/mnt/c/Users/LENOVO/Documents/Foundry/Fund-ME/test/FundMe.t.sol`

Context:
- "/mnt/c/Users/LENOVO/Documents/Foundry/Fund-ME/test/FundMe.t.sol": No such file or directory (os error 2)
make: *** [Makefile:7: deploy-sepolia] Error 1
suhas@DESKTOP-8DHC3LB:/mnt/c/Users/LENOVO/Documents/Foundry/Fund-ME$

help where i can modify the source path

**Comment by EngrPips** [comment]
Please show your makeFile

**Reply by Suhas-30** [reply]
-include .env

build:; forge build

deploy-sepolia:

	forge script script/DepolyFundMe.s.sol:DeployFundMe --rpc-url $(SEPOLIA_RPC_URL) --private-key $(PRIVATE_KEY) --broadcast --verify --etherscan-api-key $(ETHERSCAN_API_KEY) -vvvv

above is the makeFile

**Reply by EngrPips** [reply]
<img width="308" alt="Screenshot 2024-08-15 at 15 21 34" src="https://github.com/user-attachments/assets/dc6c3c07-480d-49b8-9382-2c970eed8932">

Show your working directory like I showed mine above. Please open the script folder so that we can see what is inside.

**Reply by EngrPips** [reply]
Also, are you running a script or test file because your title doesn't match the error in the body of your message.

**Reply by Suhas-30** [reply]
![Uploading Screenshot 2024-08-15 195521.png…]()

**Reply by EngrPips** [reply]
Your image does not load. What command did you run before you ran into this issue?

**Reply by Suhas-30** [reply]
![Screenshot 2024-08-15 195521](https://github.com/user-attachments/assets/6c37b2c7-44fa-4abe-b1b4-570e0dd32f83)

**Reply by Suhas-30** [reply]
can you figure it now?

**Reply by Suhas-30** [reply]
this cmd i ran make deploy-sepolia

**Reply by EngrPips** [reply]
What command did you run before you ran into this issue?

**Reply by EngrPips** [reply]
How are you getting an error related to the test file ` error="/mnt/c/Users/LENOVO/Documents/Foundry/Fund-ME/test/FundMe.t.sol` if you ran `deploy-sepolia`?

**Reply by Suhas-30** [reply]
make deploy-sepolia

**Reply by EngrPips** [reply]
This command `forge script script/DepolyFundMe.s.sol:DeployFundMe --rpc-url $(SEPOLIA_RPC_URL) --private-key $(PRIVATE_KEY) --broadcast --verify --etherscan-api-key $(ETHERSCAN_API_KEY) -vvvv` have nothing to do with the test file.

**Reply by EngrPips** [reply]
Maybe run `foundryup` and retry because it is weird that you ran `make deploy-sepolia` and you are running into an error with the test file

**Reply by Suhas-30** [reply]
still i'm getting error

**Reply by EngrPips** [reply]
Show your teminal

**Reply by EngrPips** [reply]
<img width="1440" alt="Screenshot 2024-08-15 at 15 39 12" src="https://github.com/user-attachments/assets/92956d00-9d24-4450-8745-43700f6ebf3b">

The way I showed mine above

**Reply by Suhas-30** [reply]
![Screenshot 2024-08-15 200859](https://github.com/user-attachments/assets/90f70022-59f0-4412-b13d-89ffd51230a6)
![Screenshot 2024-08-15 200942](https://github.com/user-attachments/assets/0b000347-988f-4c27-8c1e-1f70db6056ab)

**Reply by Suhas-30** [reply]
![Screenshot 2024-08-15 201259](https://github.com/user-attachments/assets/b75e0e9c-c7a2-4500-a392-0c3404887aa1)

**Reply by Suhas-30** [reply]
?

**Reply by EngrPips** [reply]
I am unsure what the problem is here. It seems like you are on `wsl`, right?

**Reply by Suhas-30** [reply]
yes

**Reply by EngrPips** [reply, marked as answer]
run `foundryup` then `forge clean` then retry the process.

**Reply by Suhas-30** [reply]
Hey Thanks it's working. can you tell me what happend?

**Reply by EngrPips** [reply]
It seems like stuff didn't compile correctly initially, and it cached the first process and then used it for every other process, but when you ran `forge clean` it basically cleared any cache and ran a fresh process.

**Reply by xueyuanhuang** [reply]
hey, thanks, I have the same issue too! Maybe the video should clarify this, as I think most people who follow the video will have this issue.

**Reply by EngrPips** [reply]
Yeah, I get your point but this was not a thing when the course was made. Definitely some kind of awareness would be great.

**Reply by xueyuanhuang** [reply]
Might be right, from my view this is really like  scenario where in remix you have to create a folder contracts so your compile in zksync will success.

People shouldn't waste time for this.

**Reply by EngrPips** [reply]
Exactly, Not everything you can completely achieve in one go, some problems you leave and come back to them.

**Comment by SuruiLiu** [comment]
Mark! Meet this problem too.

**Reply by EngrPips** [reply]
Interesting.
